### ISSUE TITLE
The method 'ancestorWidgetOfExactType' isn't defined for the class 'BuildContext'.

### ISSUE THREAD

**Issue by bastaware** [mobiten/flutter_staggered_animations#29]
../../../../../flutter/.pub-cache/hosted/pub.dartlang.org/flutter_staggered_animations-0.1.2/lib/src/animation_configuration.dart:161:20: Error: The method 'ancestorWidgetOfExactType' isn't defined for the class 'BuildContext'.
 - 'BuildContext' is from 'package:flutter/src/widgets/framework.dart' ('../../../../../flutter/packages/flutter/lib/src/widgets/framework.dart').
Try correcting the name to the name of an existing method, or defining a method named 'ancestorWidgetOfExactType'.
    return context.ancestorWidgetOfExactType(AnimationConfiguration)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^

Seems to be deprecated on master.

https://stackoverflow.com/questions/65657716/error-the-method-ancestorstateoftype-isnt-defined-for-the-class-buildcontex

**Comment by josh-burton** [comment]
Now deprecated in beta too

**Comment by movieator** [comment]
Same issue on dev channel. Can you fix this please?

**Comment by real1900** [comment]
Having the same issue.... Pllllleeeeease fix

**Comment by nilsreichardt** [comment]
There is already a PR: https://github.com/mobiten/flutter_staggered_animations/pull/30

Until #30 isn't merged, this can be used `pubspec.yaml`:
```
dependencies:
  flutter_staggered_animations:
    git:
      url: https://github.com/burton/flutter_staggered_animations.git
      ref: findAncestorWidgetOfExactType
```

**Comment by cyberpwnn** [comment]
> There is already a PR: #30
> 
> Until #30 isn't merged, this can be used `pubspec.yaml`:
> 
> ```
> dependencies:
>   flutter_staggered_animations:
>     git:
>       url: https://github.com/burton/flutter_staggered_animations.git
>       ref: findAncestorWidgetOfExactType
> ```

I think your repo is private.

**Comment by nilsreichardt** [comment]
@cyberpwnn There was a mistake in my answer :| I fixed it :)

**Comment by real1900** [comment]
You mean until it is merged , not until it isn't merged.

**Comment by nilsreichardt** [comment]
@real1900 oh, yes! Thanks!

**Comment by azack** [comment]
@CallMeSH This is going to be a clocking issue for people

**Comment by MrLepage** [comment]
We have planned to fix this next week.

**Comment by nilsreichardt** [comment]
> We have planned to fix this next week.

There is already a PR which fix this issue: #30

**Comment by nilsreichardt** [comment]
#30 ist merged. This issue can be closed :)
